object/interp.go

```diff
@@ -26,6 +44,19 @@ func CreateFunction(cmd Extension) error {
 	if cmd.Name == "" {
 		return errors.New("empty command name")
 	}
+	// Only support 1 level of namespace for now.
+	dotSplit := strings.SplitN(cmd.Name, ".", 2)
```

Comment:
This is the kind of thing I'm a bit afraid of

I would expect a method to do the magic of
- encode and do [this](https://github.com/grol-io/grol/pull/217/files#diff-f80b989da1d1c5f67f5ac9041afd2d10aa0810075359ff3109a92f571ef82762R299) 
- and one to split

This way the code is easy to use and if something changes later, it avoids issues

Reply:
it's just a split by first/only 1 allowed `.` though (and it's covered by the e2e tests) the code you linked in particular needs to stay minimal/inline because of the cost and where it is in code paths